server/src/main/resources/db/changelog/20191009160315-remove-duplicate-indexes.xml

```diff
@@ -0,0 +1,35 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<!--
+  ~  Copyright (c) 2009 - ${YEAR} Red Hat, Inc.
+  ~
+  ~  This software is licensed to you under the GNU General Public License,
+  ~  version 2 (GPLv2). There is NO WARRANTY for this software, express or
+  ~  implied, including the implied warranties of MERCHANTABILITY or FITNESS
+  ~  FOR A PARTICULAR PURPOSE. You should have received a copy of GPLv2
+  ~  along with this software; if not, see
+  ~  http://www.gnu.org/licenses/old-licenses/gpl-2.0.txt.
+  ~
+  ~  Red Hat trademarks are not licensed under GPLv2. No permission is
+  ~  granted to use or replicate Red Hat trademarks that are incorporated
+  ~  in this software or its documentation.
+  -->
+
+<databaseChangeLog
+        xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+        http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.1.xsd">
+
+
+    <changeSet id="20191009160315-1" author="sdhome" dbms="postgresql">
```

Comment:
Is there a reason that this changeset is done with dbms set to postgres only, while the second is for all 3 (mysql,mariadb,postgresql)?

Reply:
The index doesn't exist in the MySQL/MariaDB version:

```
candlepin=# \d cp_dist_version
                      Table "public.cp_dist_version"
    Column    |           Type           | Collation | Nullable | Default 
--------------+--------------------------+-----------+----------+---------
 id           | character varying(32)    |           | not null | 
 name         | character varying(255)   |           | not null | 
 display_name | character varying(255)   |           | not null | 
 created      | timestamp with time zone |           |          | 
 updated      | timestamp with time zone |           |          | 
Indexes:
    "cp_dist_version_pkey" PRIMARY KEY, btree (id)
    "cp_dist_version_name_key" UNIQUE CONSTRAINT, btree (name)
    "cp_dist_version_id_idx" btree (id)
Referenced by:
    TABLE "cp_dist_version_capability" CONSTRAINT "fk_dist_vrsn_capability" FOREIGN KEY (dist_version_id) REFERENCES cp_dist_version(id)




MariaDB [candlepin]> SHOW INDEX FROM cp_dist_version;
+-----------------+------------+--------------------------+--------------+-------------+-----------+-------------+----------+--------+------+------------+---------+---------------+
| Table           | Non_unique | Key_name                 | Seq_in_index | Column_name | Collation | Cardinality | Sub_part | Packed | Null | Index_type | Comment | Index_comment |
+-----------------+------------+--------------------------+--------------+-------------+-----------+-------------+----------+--------+------+------------+---------+---------------+
| cp_dist_version |          0 | PRIMARY                  |            1 | id          | A         |           5 |     NULL | NULL   |      | BTREE      |         |               |
| cp_dist_version |          0 | cp_dist_version_name_key |            1 | name        | A         |           5 |     NULL | NULL   |      | BTREE      |         |               |
+-----------------+------------+--------------------------+--------------+-------------+-----------+-------------+----------+--------+------+------------+---------+---------------+
2 rows in set (0.001 sec)

```